db/migrate/20220507202056_add_default_value_to_predefined_attribute.rb

```diff
@@ -0,0 +1,7 @@
+class AddDefaultValueToPredefinedAttribute < ActiveRecord::Migration[6.1]
+  def change
+    def up
+      change_column :expences, :predefined, :boolean, default: false
+    end
```

Comment:
optional: 
add 'down' logic

Reply:
I dont see a sense in down logic in this case, probably I should not make up function